packages/api/src/controllers/generate.test.ts

```diff
@@ -260,7 +270,7 @@ describe("controllers/generate", () => {
 
     for (const [title, input, error] of testCases) {
       it(title, async () => {
-        const res = await client.fetch("/beta/generate/image-to-image", {
+        const res = await client.fetch("/generate/image-to-image", {
```

Comment:
Why are these (everything below) not using the base path?

Reply:
Yeah if we have some of the tests testing both, let's be consistent. I'm fine with just testing the new one as well and leaving the old untested. But we can do that when we remove the `/beta`/ ones